### ISSUE TITLE
Contains do not work in queries in Mockup

### ISSUE THREAD

**Issue by krihvi** [delegateas/XrmMockup#72]
Contains is supported as a [filter in linq queries to CRM](https://docs.microsoft.com/en-us/dynamics365/customer-engagement/developer/webapi/query-data-web-api)

The following example tries to get the list of Accounts, which name contains an "a". An Account has been added the fits the criteria. 

```csharp 
[TestMethod]
public void GitHubIssueTest()
{
    var account = new Account()
    {
        Name = "TestAcca",

    };
    orgAdminService.Create(account);

    var searchstring = "a";
    using (var xrm = new Xrm(orgAdminService))
    {
        var query_contains1 = (from c in xrm.AccountSet
                               where c.Name.Contains(searchstring)
                               select new
                               {
                                   c.Name
                               }).ToList();
        tracingService.Trace("Number of Accounts containing a" + query_contains1.Count);
        Assert.IsTrue(query_contains1.Count > 0);
    }
}
```
The Test fails.
query_contains1 returns an empty list in mockup, but works in CRM.
query_contains1 do not throw an exception.

**Comment by TomMalow** [comment]
Not contains also seems to be a valid filtering now https://docs.microsoft.com/en-us/dynamics365/customer-engagement/developer/org-service/linq-query-examples#use-the-does-not-contain-operator
So it would make sense to add this as well

**Comment by magesoe** [comment]
Fixed in 0.10.1
